client/src/main/model/Result.ts

```diff
@@ -1,7 +1,7 @@
 export default interface Result {
-  end: Date;
+  end: string;
   rank: number;
-  start: Date;
+  start: string;
```

Comment:
Why?

Reply:
Just making typescript happy. Also, this is display only.

![image](https://github.com/thewca/statistics/assets/10170850/42b3db61-939d-4234-9433-2a7ae6530644)